### DISCUSSION TITLE
整个绳网没有比我更垃圾的显卡！

### DISCUSSION THREAD

**Discussion by Ziew-Ryan** [share121/inter-knot#2741] [其他]
CPU:i5-9400
GPU:NVIDIA Geforce GT 730/Intel UHD Graphics 630
RAM:8GB
欢迎挑战！

**Comment by YuShaozhu** [comment]
cpu：i3 2120
gpu：NVIDIA Geforce 8600GT
ram：4GB

**Reply by Ziew-Ryan** [reply]
哇……

**Comment by lllliufeng** [comment]
我GF210不服

**Reply by Ziew-Ryan** [reply]
这是哪一年的了😱

**Comment by LuAn-afl** [comment]
我手机不服

**Reply by cpu644** [reply]
有一说一骁龙8gen2转译windows性能也能拼他那个九代i5了

**Reply by Ziew-Ryan** [reply]
8Gen2以上就应该能拼一拼

**Comment by cpu644** [comment]
![屏幕截图 2024-08-14 215102](https://github.com/user-attachments/assets/3bbc0fb0-07b1-454b-bf9c-e4879f16440a)
gt610 10gb 限定版
r5 1600x 8核@4.8ghz 限定版
2*24gb ddr3

**Reply by Ziew-Ryan** [reply]
6

**Comment by yuxigongzi** [comment]
轻薄本，MX350，显存2G

**Reply by cpu644** [reply]
350你搁现在还能用dx12呢。fsr一开还能720p玩点3a

**Reply by Ziew-Ryan** [reply]
蛙趣

**Comment by hktklml** [comment]
UHD Graphics 600 250MHZ，赛扬J4125

**Reply by Ziew-Ryan** [reply]
查了一下，19年第四季度发布，比我19年第一季度新一点

**Reply by hktklml** [reply]
不不不，这是一个单核150，多核500的U，显卡只能放视频，游戏只有15帧

**Comment by LT-UT** [comment]
<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >
建议花一百多在闲鱼买个1030

**Reply by Ziew-Ryan** [reply]
我还是个5年纪学生，电脑全家公用，想换显卡也换不了啊😭

**Comment by ArcticOrigin** [comment]
![QQ截图20240819105822](https://github.com/user-attachments/assets/24b09fd3-2759-4f19-a776-310af54ccd74)
?

**Reply by Ziew-Ryan** [reply]
张大嘴巴.gif

**Comment by realyniger** [comment]
有一说一，邦布内置的ai算力卡拿来打游戏都比你这好...楼主还是考虑换机子吧。

**Reply by Ziew-Ryan** [reply]
没办法啊，我还是个5年纪学生，电脑全家公用，想换也换不了啊😭

**Reply by LT-UT** [reply]
如果你有钱，父母又同意，那就买一个外接显卡，但很贵

**Comment by Goodname-LOL** [comment]
切~我直接用核显(￣▽￣)╭
![image](https://github.com/user-attachments/assets/5d1c2e9e-7083-423d-bd05-0953ff558e19)

**Comment by x97-1** [comment]
##### mac有显卡吗

**Comment by L-walnut** [comment]
我用的3050ti laptop
<img src="https://img-static.mihoyo.com/communityweb/upload/70b5b0c9543ee5080c6ae92fd2797892.png" alt="心情复杂" width="75" >

**Reply by Fintbo** [reply]
同3050Laptop 纯纯坐牢显卡<img src="https://img-static.mihoyo.com/communityweb/upload/1b5600e9d6d16de86447348909796aa4.png" alt="安比 哭哭" width="75" >

**Comment by fengyunLUFOG** [comment]
CPU：Intel Core i5 9400F
GPU:NVIDIA GeForce GT 710
RAM:8GB
